ocsci/pytest_customization/ocscilib.py

```diff
@@ -70,6 +82,32 @@ def pytest_configure(config):
     """
     if not config.getoption("--help"):
         process_cluster_cli_params(config)
+        # Add OCS related versions to the html report and remove extraneous metadata
+        print("Fetching Cluster versions .....")
```

Comment:
What is the reason to use `print(...)` instead of logging e.g. `log.info(...)`?

Reply:
somehow when I use logging module, it throws error. I would like to take that part in next PR if thats OK